src/entrypoint.sh

```diff
@@ -12,9 +12,9 @@ fi
 ## GCS bucket is mounted by chart
 # We can just move the artifacts
 if [ "$DO_UPLOAD" -eq "1" ]; then
-    mv /backups/output/* /mnt/backup-bucket/
+    ./upload.sh
 else
-    echo "Skip uploading..."
+    echo "Skipping upload as DO_UPLOAD is not set..."
```

Comment:
We could also remove this option now as we can also "upload" in local environments.